README.md

```diff
@@ -0,0 +1,34 @@
+fdb-exporter
+============
+
+Exports a subset of FoundationDB's "status json" command output as prometheus metrics using tally. It uses the FoundationDB client to connect to the cluster. 
+
+Configuration
+-------------
+
+This exporter is meant to be run in containers (it can be run a standlone service too, but no init scripts or systemd manifests are provided). The latest container image can be found at `tigrisdata/fdb-exporter:main`. 
+
+The configuration is done through the following environment variables:
+
+* `FDB_CLUSTER_FILE`: the cluster file to connect to FoundationDB
+* `FDB_API_VERSION`: the api version to use for the FoundationDB connection (default: 710)
+* `ENVIRONMENT`: exposed directly as the `environment` tag in all metrics
+* `SERVICE`: exposed directly as the `service` tag in all metrics
+* `FDB_VERSION`: exposed directly as the `version` tag in all metrics
+* `FDB_CLUSTER_NAME`: exposed directly as the `cluster` tag in all metrics
+* `FDB_EXPORTER_HTTP_LISTEN_ADDR`: the address where the built-in web server will listen on and expose the metrics on `/metrics` url. The default is `:8080`
+
+Building
+--------
+
+To build a binary, simple run `make`. This will put an `fdb-exporter` binary in the current working directory. To make a contriner image, run `docker build -t YOUR_ORG_HERE/fdb-exporter:YOUR_TAG_HERE .`.
```

Comment:
You probably could remove `simple` or change to `simply`

Reply:
Sorry also typo on `container`

Reply:
Great catch, thanks!